src/audio/Kconfig

```diff
@@ -134,6 +134,16 @@ config CORDIC_FIXED
 	  Start with the angle in the range [-2*pi, 2*pi) and output has range in
 	  [-1.0 to 1.0]
 
+config ARITHMETIC_FIXED
```

Comment:
@singalsu should we have this config labelled per are i.e. trig, log etc ? or per unit type trig_fixed, log_fixed.

Reply:
It looks like this patch could be split to LOG_FIXED and POWER_FIXED. If only one needed fair amount of tables RAM would be saved.

Reply:
Do i need any action here ? if yes, please let me know. Thank you

Reply:
Yes please, can you split ARITHMETIC_FIXED into log and power parts in kconfig if they can be provided separately with saved tables RAM.